h/schemas/forms/accounts/reset_password.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import colander
+import deform
+from itsdangerous import BadData, SignatureExpired
+
+from h import i18n, models
+from h.schemas.base import CSRFSchema
+from h.schemas.forms.accounts import util
+
+_ = i18n.TranslationString
+
+PASSWORD_MIN_LENGTH = 2  # FIXME: this is ridiculous
+USERNAME_BLACKLIST = None
```

Comment:
Both of these constants appear to be unused in this file. Additionally `USERNAME_BLACKLIST` is mutated in `h/accounts/schemas.py` by the `get_blacklist` function so avoid duplicating that!

Reply:
Great catch! Thanks!